src/objclass/class_api.cc

```diff
@@ -17,14 +17,14 @@ static constexpr int dout_subsys = ceph_subsys_objclass;
 
 static ClassHandler *ch;
 
-void cls_initialize(ClassHandler *h)
+void cls_host_initialize(ClassHandler *h)
```

Comment:
i'd suggest squash this commit into "objclass, osd: turn ClassHandler into singleton." and add a commit to drop "cls_finalize()" before "objclass, osd: turn ClassHandler into singleton." for emphasize that that function is not used anywhere.

Reply:
Done.